lib/music_album.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'item'
+
+class MusicAlbum < Item
+  def initialize(on_spotify, publish_date, archived, id: Random.rand(1..1000))
```

Comment:
Kindly remove the id property from the constructor since it should not be used as the parameter of the constructor per the class diagram 😉